playbooks/besman-scorecard-playbook.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# Author: Samir Ranjan Parhi
+# License: Same as Repository Licence
+# Usage : Beta-Release
+# Date : 05-march-2024
+
+set -e
+RED=$(tput setaf 1)
+GREEN=$(tput setaf 2)
+YELLOW=$(tput setaf 3)
+BLUE=$(tput setaf 4)
+RESET=$(tput sgr0)
+BOLD=$(tput bold)
+
+echo -e "\n${BLUE}###################################################${RESET}"
+
+echo -e "\n${BOLD}${GREEN}Info!${RESET} ${BLUE}You are using CodeQL playbook! \nSend your thoughts we would love the enhancement Ideas.\n${RESET}"
+
+echo -e "\n${BLUE}###################################################${RESET}"
+
+    PROJECT_NAME=$1
+    github_token=$2
+    github_api_version=$3
+    github_project_owner=$4
+    github_repo_name=$5
+    project_version=$6  
+
+# Lifecycle functions
+function __besman_init {
+
+    echo -e "\n${BLUE}🛫 Preparing your machine for ScoreCard Run .${RESET}"
+    # Specify the directory to save the scan results
+    mkdir -p "$HOME/$PROJECT_NAME/scoreCard"
+
+}
+
+function __besman_execute {
+
+    # creating a CodeQL database for Java code
+    curl -X 'GET' \
```

Comment:
@samirparhi-dev for this scorecard playbook to work, github actions for scorecard has to be run

Reply:
@sand-hya : Noted